packages/webapp/src/components/Form/CompositionInputs/styles.module.scss

```diff
@@ -41,44 +41,50 @@
       color: var(--Colors-Neutral-Neutral-300);
       @include truncateText();
     }
+  }
 
-    .selectWrapper {
-      border-left: none;
-    }
+  &:not(.disabled) .selectWrapper {
+    margin-right: -8px;
+    border-left: 1px solid var(--grey400);
```

Comment:
Sorry could you remind me (it feels like maybe something that we touched on in grooming a few weeks ago?) that we ended up having this line to the left of the unit? Was it to make it more consistent with the `<Unit />` component?

Reply:
I thought I would need to add a border to make it consistent... but maybe it was the white background...
![Screenshot 2024-06-19 at 1 59 45 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/8a479fa6-1ff7-441d-b098-39bf1d4eba09)
https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1512308854/S86+Grooming